server/db/schema/01_users.sql

```diff
@@ -0,0 +1,43 @@
+DROP TABLE IF EXISTS parents CASCADE;
+DROP TABLE IF EXISTS baby CASCADE;
+DROP TABLE IF EXISTS parent_baby CASCADE;
+DROP TABLE IF EXISTS LOGS CASCADE;
+
+
+CREATE TABLE parents (
+  parent_id SERIAL PRIMARY KEY NOT NULL,
+  first_name VARCHAR(255) NOT NULL,
+  last_name VARCHAR(255),
+  email_id VARCHAR(255) NOT NULL UNIQUE,
+  is_authenticated BOOLEAN DEFAULT false
```

Comment:
@mhakbar remove `is_authenticated`. authentication is handled by the login route.